pkg/terraform/workspace.go

```diff
@@ -176,16 +186,16 @@ func (w *Workspace) DestroyAsync(callback CallbackFn) error {
 		cmd.SetEnv(append(os.Environ(), w.env...))
 		cmd.SetDir(w.dir)
 		out, err := cmd.CombinedOutput()
+		if err != nil {
```

Comment:
I think we are converting a possible error from CombinedOutput to the format of the Destroy error before passing it to the callback function. This seems like an improvement to the logs.

Reply:
Hi @sergenyalcin,
Yes, correct. As in the synchronous invocations of these commands, we are transforming the errors so that we do not accidentally leak sensitive info:
https://github.com/upbound/upjet/blob/aeb5f15054f18d0eff52a75e71ec0f562fb79ffb/pkg/terraform/workspace.go#L144

These changes improve the readability in my opinion.